settings.gradle

```diff
@@ -6,11 +6,15 @@ pluginManagement {
     }
 }
 
-rootProject.name = 'tia-tutorial'
+plugins {
+    id 'org.gradle.toolchains.foojay-resolver-convention' version '0.8.0'
```

Comment:
What's this? Auto-download in case the user does not have the required JVM installed?

Reply:
Yes, relying on the installed JVM to be picked up is being removed in Gradle 9.0.

Reply:
👍